epfast-annotations/src/test/java/com/exactpro/epfast/converter/ConverterTest.java

```diff
@@ -502,13 +502,13 @@ void stringToLong() {
 
             () -> assertThatExceptionOfType(NumberFormatException.class)
                 .isThrownBy(() -> converter.stringToLong("1.7"))
-                .withMessageMatching(wordPattern("1.7"))
-                .withMessageMatching(wordPattern("long")),
+                .withMessageContaining("1.7"),
```

Comment:
The reason I don't wan't to use `withMessageContaining()` is that "11.71" will pass such test. `wordPattern()` may not cover all scenarios correctly. Mind that as well.